<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyTicketDialog</class>
 <widget class="QDialog" name="BuyTicketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>317</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Покупка билета</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="loginLabel">
       <property name="text">
        <string>Логин:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="surnameLabel">
       <property name="text">
        <string>Фамилия:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Имя:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="midnameLabel">
       <property name="text">
        <string>Отчество:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="passportSerialLabel">
       <property name="text">
        <string>Серия паспорта:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="passportNumberLabel">
       <property name="text">
        <string>Номер паспорта:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="dateDepartureLabel">
       <property name="text">
        <string>Дата вылета:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="loginLine"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="surnameLine"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="nameLine"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="midnameLine"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="passportSerialLine"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="passportNumberLine"/>
     </item>
     <item row="6" column="1">
      <widget class="QDateEdit" name="dateDepartureLine"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="errorLabel">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="rejectButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="text">
        <string>Принять</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
